pkgs/applications/graphics/renderdoc/default.nix

```diff
@@ -1,32 +1,34 @@
-{ stdenv, fetchFromGitHub, cmake, pkgconfig
+{ stdenv, fetchFromGitHub, cmake, pkgconfig, mkDerivation
 , qtbase, qtx11extras, qtsvg, makeWrapper
-, vulkan-loader, xorg
-, python3, bison, pcre, automake, autoconf
+, vulkan-loader, xorg, python3, python3Packages
+, bison, pcre, automake, autoconf, addOpenGLRunpath
 }:
 let
   custom_swig = fetchFromGitHub {
     owner = "baldurk";
     repo = "swig";
-    rev = "renderdoc-modified-6";
-    sha256 = "00ykqlzx1k9iwqjlc54kfch7cnzsj53hxn7ql70dj3rxqzrnadc0";
+    rev = "renderdoc-modified-7";
+    sha256 = "15r2m5kcs0id64pa2fsw58qll3jyh71jzc04wy20pgsh2326zis6";
   };
+  pythonPackages = python3Packages;
 in
 stdenv.mkDerivation rec {
```

Comment:
`s/stdenv.mkDerivation/mkDerivation/`

see [the manual](https://hydra.nixos.org/build/96804884/download/1/nixpkgs/manual.html#sec-language-qt)

The build otherwise fails for me with `/nix/store/[...]-stdenv-linux/setup: line 97: wrapQtApp: command not found`

Reply:
Fixed